doctl-kubernetes-cluster: add page (#8846)
tldr-pages/tldr: pages/common/doctl-kubernetes-cluster.md

# doctl kubernetes cluster

> > For managing Kubernetes clusters and viewing configuration options relating to clusters.

Reviewer: ```suggestion
> Manage Kubernetes clusters and view configuration options relating to clusters.
```

> More information: <https://docs.digitalocean.com/reference/doctl/reference/kubernetes/cluster/>.

- Create a Kubernetes cluster:

> `doctl kubernetes cluster create --count {{3}} --region {{nyc1}} --size {{s-1vcpu-2gb}} --version {{latest}} {{cluster-name}}`

Reviewer: ```suggestion
`doctl kubernetes cluster create --count {{3}} --region {{nyc1}} --size {{s-1vcpu-2gb}} --version {{latest}} {{cluster_name}}`
```

- List all Kubernetes clusters:

`doctl kubernetes cluster list`

- Fetch and save the kubeconfig:

> `doctl kubernetes cluster kubeconfig save {{cluster-name}}`

Reviewer: ```suggestion
`doctl kubernetes cluster kubeconfig save {{cluster_name}}`
```

- Check for available upgrades:

> `doctl kubernetes cluster get-upgrades {{cluster-name}}`

Reviewer: ```suggestion
`doctl kubernetes cluster get-upgrades {{cluster_name}}`
```

> - Upgrade a cluster:

Reviewer: What upgrades?

> `doctl kubernetes cluster upgrade {{cluster-name}}`

Reviewer: ```suggestion
`doctl kubernetes cluster upgrade {{cluster_name}}`
```

- Delete a cluster:

> `doctl kubernetes cluster delete {{cluster-name}}`

Reviewer: ```suggestion
`doctl kubernetes cluster delete {{cluster_name}}`
```
